src/Microsoft.Windows.CsWin32/templates/RECT.cs

```diff
@@ -0,0 +1,26 @@
+﻿partial struct RECT
+{
+	internal RECT(global::System.Drawing.Rectangle value) :
+		this(value.Left, value.Top, value.Right, value.Bottom) { }
+	internal RECT(global::System.Drawing.Point location, global::System.Drawing.Size size) :
+		this(location.X, location.Y, unchecked(location.X + size.Width), unchecked(location.Y + size.Height)) { }
+	internal RECT(int left, int top, int right, int bottom)
+	{
+		this.left = left;
+		this.top = top;
+		this.right = right;
+		this.bottom = bottom;
+	}
+
+	public static RECT FromXYWH(int x, int y, int width, int height) =>
```

Comment:
Please make all non-operator members `internal`. Tey will be promoted to `public` if appropriate. Only operators are `public`, since they must be per the C# language.